### PR TITLE
freerdp 2.0.0

### PR DESCRIPTION
**Pull request by Bo98**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/freerdp.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Freerdp < Formula
   desc "X11 implementation of the Remote Desktop Protocol (RDP)"
   homepage "https://www.freerdp.com/"
-  url "https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz"
-  sha256 "3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49"
+  url "https://pub.freerdp.com/releases/freerdp-2.0.0.tar.gz"
```
Comment (SMillerDev) [inline Formula/freerdp.rb:4]: Why is it moving?

Comment (Bo98) [reply Formula/freerdp.rb:4]: It prpobably should have been from here in the first place, but there wasn't clear directions until now.

The release page says:

> See https://pub.freerdp.com/releases/ for the release tarball.

**Comment by EricFromCanada** [comment]
```
/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang  -Wno-unused-parameter -Wno-unused-macros -Wno-padded -Wno-c11-extensions -Wno-gnu -Wno-unused-command-line-argument -Wno-deprecated-declarations -DWINPR_DLL -DNDEBUG -Wno-builtin-macro-redefined -D__FILE__='"$(subst /tmp/freerdp-20200412-95036-jty7ow/freerdp-2.0.0/,,$(subst /tmp/freerdp-20200412-95036-jty7ow/freerdp-2.0.0/,,$(abspath $<)))"' -dynamiclib -Wl,-headerpad_max_install_names -compatibility_version 2.0.0 -current_version 2.0.0 -o libfreerdp-client2.2.0.0.dylib -install_name @rpath/libfreerdp-client2.2.dylib CMakeFiles/freerdp-client.dir/client.c.o CMakeFiles/freerdp-client.dir/cmdline.c.o CMakeFiles/freerdp-client.dir/compatibility.c.o CMakeFiles/freerdp-client.dir/file.c.o CMakeFiles/freerdp-client.dir/__/__/channels/client/tables.c.o CMakeFiles/freerdp-client.dir/__/__/channels/client/addin.c.o  -Wl,-rpath,/tmp/freerdp-20200412-95036-jty7ow/freerdp-2.0.0/libfreerdp -Wl,-rpath,/tmp/freerdp-20200412-95036-jty7ow/freerdp-2.0.0/winpr/libwinpr ../../channels/drdynvc/client/libdrdynvc-client.a ../../channels/remdesk/client/libremdesk-client.a ../../channels/rdpsnd/client/librdpsnd-client.a ../../channels/rdpdr/client/librdpdr-client.a ../../channels/rdp2tcp/client/librdp2tcp-client.a ../../channels/rail/client/librail-client.a ../../channels/encomsp/client/libencomsp-client.a ../../channels/cliprdr/client/libcliprdr-client.a ../../channels/video/client/libvideo-client.a ../../channels/urbdrc/client/liburbdrc-client.a ../../channels/rdpsnd/client/librdpsnd-client.a ../../channels/rdpgfx/client/librdpgfx-client.a ../../channels/rdpei/client/librdpei-client.a ../../channels/geometry/client/libgeometry-client.a ../../channels/echo/client/libecho-client.a ../../channels/disp/client/libdisp-client.a ../../channels/audin/client/libaudin-client.a ../../channels/smartcard/client/libsmartcard-client.a ../../channels/serial/client/libserial-client.a ../../channels/printer/client/libprinter-client.a ../../channels/parallel/client/libparallel-client.a ../../channels/drive/client/libdrive-client.a ../../channels/urbdrc/client/libusb/liburbdrc-client-libusb.a ../../channels/rdpsnd/client/mac/librdpsnd-client-mac.a ../../channels/rdpsnd/client/fake/librdpsnd-client-fake.a ../../channels/printer/client/cups/libprinter-client-cups.a ../../channels/audin/client/mac/libaudin-client-mac.a -framework CoreServices ../../channels/urbdrc/client/liburbdrc-client.a /usr/local/lib/libusb-1.0.dylib -framework AVFoundation -framework CoreFoundation /usr/lib/libcups.dylib ../../libfreerdp/libfreerdp2.2.0.0.dylib ../../winpr/libwinpr/libwinpr2.2.0.0.dylib -framework AudioToolbox -framework CoreAudio -framework ApplicationServices 
Undefined symbols for architecture x86_64:
  "_objc_msgSend", referenced from:
      _rdpsnd_mac_open in librdpsnd-client-mac.a(rdpsnd_mac.m.o)
      _rdpsnd_mac_set_volume in librdpsnd-client-mac.a(rdpsnd_mac.m.o)
      _rdpsnd_mac_play in librdpsnd-client-mac.a(rdpsnd_mac.m.o)
      _rdpsnd_mac_close in librdpsnd-client-mac.a(rdpsnd_mac.m.o)
```

**Comment by Bo98** [comment]
I suspect it might be something to do with not linking with Foundation.framework or something.

**Comment by ryan-robeson** [comment]
I was able to get this to compile with:
```
inreplace "channels/rdpsnd/client/mac/CMakeLists.txt",
          "FIND_LIBRARY(CORE_FOUNDATION CoreFoundation)",
          "FIND_LIBRARY(CORE_FOUNDATION CoreFoundation)\nFIND_LIBRARY(COCOA Cocoa)"
inreplace "channels/rdpsnd/client/mac/CMakeLists.txt", "${CORE_FOUNDATION}",
          "${CORE_FOUNDATION} ${COCOA}"
```
Not sure that's the proper solution, but it seems to work.

I also had to specify the `DYLD_LIBRARY_PATH` when running `xfreerdp` (`DYLD_LIBRARY_PATH=... xfreerdp`). That's probably an issue with my current setup though.

**Comment by EricFromCanada** [comment]
Obviated by #55056.

**Comment by EricFromCanada** [comment]
Obviated by #55056.

**Pull request closed without merging**